### PR TITLE
Released chart 1.3.0

### PR DESCRIPTION
**Pull request by daniel-jeremia**
- Added url-inspector service

### CODE REVIEW

File: charts/datasaur/templates/config/datasaur-env-configMap.yaml [hunk 1]
```diff
@@ -12,6 +12,18 @@ data:
   LANGUAGE_TOOL_SERVICE_ENDPOINT: http://{{ .Release.Name }}-language-tool-service:8010/v2/check
   PRESIDIO_ANALYZER_ENDPOINT: http://{{ .Release.Name }}-presidio-analyzer-service:3000/analyze
   WAVEFORM_PEAKS_GENERATOR_HOST: http://{{ .Release.Name }}-audiowaveform-generator-service:5005
+  URL_INSPECTOR_INGRESS_URL: https://{{ .Values.ingress.host }}
+  URL_INSPECTOR_INGRESS_HOST: {{ .Values.ingress.host }}
+{{- if .Values.services.urlInspector.proxy }}
+  ENABLE_PROJECT_CREATION_WITH_PROXY: "true"
+  SESSION_DOMAIN: {{ .Values.ingress.host }}
+  UPLOAD_FILE_PROXY_URL: {{ .Values.ingress.host }}/api/static/proxy/upload
+  MAX_TOKENS_PER_SENTENCE: "2000"
```
Comment (tonoman3g) [inline charts/datasaur/templates/config/datasaur-env-configMap.yaml]: MAX_TOKENS_PER_SENTENCE: 3000? @daniel-jeremia

File: charts/datasaur/templates/config/datasaur-env-configMap.yaml [hunk 2]
```diff
@@ -12,6 +12,18 @@ data:
   LANGUAGE_TOOL_SERVICE_ENDPOINT: http://{{ .Release.Name }}-language-tool-service:8010/v2/check
   PRESIDIO_ANALYZER_ENDPOINT: http://{{ .Release.Name }}-presidio-analyzer-service:3000/analyze
   WAVEFORM_PEAKS_GENERATOR_HOST: http://{{ .Release.Name }}-audiowaveform-generator-service:5005
+  URL_INSPECTOR_INGRESS_URL: https://{{ .Values.ingress.host }}
+  URL_INSPECTOR_INGRESS_HOST: {{ .Values.ingress.host }}
```
Comment (tonoman3g) [inline charts/datasaur/templates/config/datasaur-env-configMap.yaml:16]: have you added rate limit specific only for proxy?

```
  upload-proxy:
  - key: RATE_LIMIT_MAX
    value: 50
  - key: RATE_LIMIT_UPLOAD_PROXY
    value: 10000
```

File: charts/datasaur/values.yaml [hunk 3]
```diff
@@ -23,6 +23,7 @@ services:
       targetMemoryUtilizationPercentage: 80
   worker:
     critical:
+      enabled: true
```
Comment (tonoman3g) [inline charts/datasaur/values.yaml:26]: have you update the setting for nginx's max file upload size?

File: charts/datasaur/templates/config/datasaur-env-configMap.yaml (hunk 1 above)
Comment (tonoman3g) [inline charts/datasaur/templates/config/datasaur-env-configMap.yaml]: oh ya, i think env var MAX_TOKENS_PER_SENTENCE is not specific for upload-proxy, so you can move it out from it

File: charts/datasaur/templates/config/datasaur-env-configMap.yaml [hunk 4]
```diff
@@ -12,6 +12,18 @@ data:
   LANGUAGE_TOOL_SERVICE_ENDPOINT: http://{{ .Release.Name }}-language-tool-service:8010/v2/check
   PRESIDIO_ANALYZER_ENDPOINT: http://{{ .Release.Name }}-presidio-analyzer-service:3000/analyze
   WAVEFORM_PEAKS_GENERATOR_HOST: http://{{ .Release.Name }}-audiowaveform-generator-service:5005
+  URL_INSPECTOR_INGRESS_URL: https://{{ .Values.ingress.host }}
```
Comment (tonoman3g) [inline charts/datasaur/templates/config/datasaur-env-configMap.yaml:15]: can we set the host to different domain? like i explained previously, the upload proxy should have different url

datasaur app -> cloudflare proxy on -> file size limitation
upload proxy -> cloudflare proxy off -> no file size limitation 

@daniel-jeremia

File: charts/datasaur/values.yaml (hunk 3 above)
Comment (daniel-jeremia) [reply charts/datasaur/values.yaml:26]: we have variable for this
`ingress.proxyBodySize`

File: charts/datasaur/templates/config/datasaur-env-configMap.yaml (hunk 4 above)
Comment (daniel-jeremia) [reply charts/datasaur/templates/config/datasaur-env-configMap.yaml:15]: should this be an option or upload-proxy must have different url? @tonoman3g

Comment (tonoman3g) [reply charts/datasaur/templates/config/datasaur-env-configMap.yaml:15]: lets continue it on another PR

**Review by tonoman3g — APPROVED** [review]

**Pull request merged**